src/universal/modules/summary/containers/MeetingSummary/MeetingSummaryContainer.js

```diff
@@ -99,20 +100,50 @@ const mapStateToProps = (state, props) => {
   };
 };
 
-const MeetingSummaryContainer = (props) => {
-  return <MeetingSummary {...props} />;
-};
+@requireAuth
+@connect(mapStateToProps)
+export default class MeetingSummaryContainer extends Component {
+  static propTypes = {
+    actionCount: PropTypes.number,
+    agendaItemsCompleted: PropTypes.number,
+    dispatch: PropTypes.func,
+    meetingNumber: PropTypes.number,
+    projectCount: PropTypes.number,
+    teamId: PropTypes.string,
+    teamMembers: PropTypes.array,
+    teamName: PropTypes.string
+  };
+
+  componentWillMount() {
+    this.state = {segmentEventTracksSent: false};
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const {segmentEventTracksSent} = this.state;
+    const {dispatch, meetingNumber} = nextProps;
+    if (!meetingNumber || segmentEventTracksSent) return;
+    /*
+     * Track meeting completitions by idenity.
+     *
+     * N.B. it is ok if these are sent as dupes, i.e. when viewed from
+     * meeting history.
+     */
+    dispatch(segmentEventTrack('Meeting Completed', { meetingNumber }));
```

Comment:
What I'm really after here is just getting the highest meeting number achieved by user, only for the next few weeks. This seemed like a good first stab at getting that data without having to also add segment to the server.

In the future (where everything is better) I can do that. Following this PR, I'll create and issue to make it so.